### ISSUE TITLE
I having a error in v0.3.4 and v0.3.5

### ISSUE THREAD

**Issue by Ktilis** [micahflee/torbrowser-launcher#654]
```ktilis@ktilis:~/Загрузки/torbrowser-launcher-0.3.4$ TBL_SHARE=share ./torbrowser-launcher
Tor Browser Launcher
By Micah Lee, licensed under MIT
version 0.3.4
https://github.com/micahflee/torbrowser-launcher
Downloading Tor Browser for the first time.
Downloading https://aus1.torproject.org/torbrowser/update_3/release/Linux_x86_64-gcc3/x/en-US
Traceback (most recent call last):
  File "/home/ktilis/Загрузки/torbrowser-launcher-0.3.4/./torbrowser-launcher", line 30, in <module>
    torbrowser_launcher.main()
  File "/home/ktilis/Загрузки/torbrowser-launcher-0.3.4/torbrowser_launcher/__init__.py", line 92, in main
    gui.move(
TypeError: arguments did not match any overloaded call:
  move(self, QPoint): argument 1 has unexpected type 'float'
  move(self, int, int): argument 1 has unexpected type 'float'
```

**Comment by ValarMarkhulis** [comment]
Found any fix yet? I have the same problem on Ubuntu 22.04

**Comment by davidbaumann** [comment]
Duplicate of #628
